### ISSUE TITLE
Missing msgid in Zone.pm

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-engine#613]
For the message (message tag) WRONG_SOA in Zone.pm the msgid is empty, which makes gettext so unhappy that it refuses to create new po files.

```
    WRONG_SOA => sub {
        __x    # WRONG_SOA
          '', @_;
    },
```

**Comment by vlevigneron** [comment]
OK, it seems I forgot a commit. Let me check.

**Comment by matsduf** [comment]
PR #614 will resolve this issue.

**Comment by matsduf** [comment]
Resolved by #614.
